docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -20,6 +20,14 @@ Please refer to the appropriate section depending on your context.
 
 GraalVM imposes a number of constraints and making your application a native executable might require a few tweaks.
 
+=== Quarkus Annotations
+
+This guide covers some of the annotations that Quarkus supports to help generate native images. In some cases these
+annotations are applied directly to a relevant class, but in others the class that they are placed on does not actually
+matter (e.g. if you are referencing a class in a 3rd party jar). In this case we recommend creating an empty `NativeConfig`
+class somewhere in your application and placing all annotation that control native image generation on it, so they are all
+in the same place.
```

Comment:
I wonder if we should have a table here summarizing the annotations so that the user has a global overview of what's possible.